server/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java

```diff
@@ -460,6 +528,13 @@ public FsInfo(StreamInput in) throws IOException {
         this.total = total();
     }
 
+    public FsInfo renderEffectiveWaterMark(final DiskThresholdSettings masterThresholdSettings) {
```

Comment:
Naming nit/suggestion

```suggestion
    public FsInfo setEffectiveWatermarks(@Nullable final DiskThresholdSettings masterThresholdSettings) {
```